applications/ParticleMechanicsApplication/custom_utilities/mpm_explicit_utilities.cpp

```diff
@@ -237,15 +237,15 @@ namespace Kratos
     /***********************************************************************************/
 
     void MPMExplicitUtilities::CalculateMUSLGridVelocity(
-        Element& rElement,
-        const Vector& rN)
+        Element& rElement)
     {
         KRATOS_TRY
 
         GeometryType& rGeom = rElement.GetGeometry();
         const SizeType dimension = rGeom.WorkingSpaceDimension();
         const SizeType number_of_nodes = rGeom.PointsNumber();
         const ProcessInfo& rProcessInfo = ProcessInfo();
+        const Matrix& r_N = rElement.GetGeometry().ShapeFunctionsValues();
```

Comment:
Not quite sure what you mean @philbucher . With the quadrature point change, the shape functions can be accessed (not created) from the element geom.